src/sinks/tsd-query-log-sink.ts

```diff
@@ -72,22 +72,25 @@ export class TsdQueryLogSink implements tsdDef.Sink {
     private static stringify(metricsEvent:tsdDef.MetricsEvent):string {
         var transformedMetricsEvent = TsdQueryLogSink.transformMetricsEvent(metricsEvent);
 
-        return JSON.stringify(utils.stenofy(transformedMetricsEvent),
+        return JSON.stringify(utils.stenofy(transformedMetricsEvent, transformedMetricsEvent.annotations["_host"]),
             (key, value) => {
                 if (tsdSink.TsdSink.isMetricSample(value)) {
-                    return {
-                        value: (<tsdDef.MetricSample>value).getValue(),
-                        unit: (<tsdDef.MetricSample>value).getUnit()
+                    // TODO(matthayter): implement unit numerators & denominators
```

Comment:
Yes, this was discussed offline. Proceed with a two-step plan to 2G; however, the intermediate commits will not be released.